src/llamacrawl/services/query/synthesis.py

```
@@ -68,21 +69,23 @@ def __init__(self, config: Config):
+                temperature=0.1,
                 max_tokens=self.max_input_tokens,
             )
 
             logger.info("AnswerSynthesizer initialized successfully")
 
         except Exception as e:
             logger.error(
-                f"Failed to initialize Claude Agent LLM: {e}",
+                f"Failed to initialize Qwen Ollama LLM: {e}",
                 extra={"error": str(e), "error_type": type(e).__name__},
             )
```

Ruff (0.14.0)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)